mesh/mesh.cpp

```diff
@@ -9570,9 +9570,11 @@ double Mesh::AggregateError(const Array<double> &elem_error,
          case 0: error = std::min(error, err_fine); break;
          case 1: error += err_fine; break;
          case 2: error = std::max(error, err_fine); break;
+         case 3: error = std::pow(err_fine, 2.0); break;
```

Comment:
Shouldn't this be rather
```suggestion
         case 3: error += std::pow(err_fine, 2.0); break;
```
And also the first error is not squared.